fontforgeexe/pythonui.c

```diff
@@ -144,6 +144,13 @@ return;
 static void cvpy_menuactivate(GWindow gw,struct gmenuitem *mi,GEvent *e) {
     CharView *cv = (CharView *) GDrawGetUserData(gw);
     PyObject *pysc = PySC_From_SC(cv->b.sc);
+    // The following two lines look like a noop but are very important. They
+    // set python_fv_object on the fv if it's not there, ensuring it's possible
+    // to do `glyph.font` on glyphs used as arguments in menus.  The pointer
+    // `fv` refers to is also accessible via cv->b.sc, as cv->b.sc->parent->fv,
+    // which is done (eventually) by PyFF_Glyph_get_font.
+    FontViewBase *fv = cv->b.fv;
+    PyObject *pyfv = PyFV_From_FV(fv);
```

Comment:
With "stronger" compiler warning flags this is likely to add an "assigned but not used" warning.

Reply:
I can remove the assignments, basically rewriting both lines as one line. Does that help?